plugins/cache-ehcache/src/main/java/org/opensearch/cache/store/disk/EhcacheDiskCache.java

```diff
@@ -444,12 +444,22 @@ public void close() {
     }
 
     /**
-     * Relevant stats for this cache.
-     * @return CacheStats
+     * Relevant stats for this cache, with no aggregation.
+     * @return ImmutableCacheStatsHolder
      */
     @Override
     public ImmutableCacheStatsHolder stats() {
-        return cacheStatsHolder.getImmutableCacheStatsHolder();
+        return stats(null);
+    }
```

Comment:
I am assuming this is convenience method for `stats(null)`. Can we provide default implementation within ICache itself to avoid unnecessary override?